src/styles/WindDirectionIcon.css

```diff
@@ -0,0 +1,4 @@
+.wind-icon {
+    width: 2vw;
+    height: 4vh;
```

Comment:
These look a bit crushed up on screen, would it not be better if they were the same width and height? Also, using the screen width and height doesnt seem like the best approach here, would be better to use px instead